src/main/java/controller/BitPay.java

```diff
@@ -33,13 +33,14 @@
 import com.fasterxml.jackson.databind.JsonMappingException;
 import com.fasterxml.jackson.databind.JsonNode;
 import com.fasterxml.jackson.databind.ObjectMapper;
-import com.google.bitcoin.core.ECKey;
+import org.bitcoinj.core.ECKey;
 
 public class BitPay {
 
     private static final String BITPAY_API_VERSION = "2.0.0";
     private static final String BITPAY_PLUGIN_INFO = "BitPay Java Client " + BITPAY_API_VERSION;
-    private static final String BITPAY_URL = "https://bitpay.com/";
+    public static final String BITPAY_URL = "https://bitpay.com/";
```

Comment:
I'm using the bitpay client as a dependency in my software where I use this string. Not a big deal, but one of the use cases of this library is to perform bitpay stuff on the server and present the invoice in the web client. For that, the invoice string needs to be constructed, and for that I need the bitpay url. I also added whether we're in test/production. No prob if you leave this out, I agree it's a quick hack, but support for building the URL should be in the library imho.